src/components/ReplyList.jsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
+import { getReplyById } from "../api";
+import styles from "../components/styles/ReplyList.module.css";
+import { Kebab } from "./Kebab";
+import { Dropdown } from "./Dropdown";
+import { formatDate } from "../utils/formDate";
+import { getHoursDiff } from "../utils/diffDate";
+
+export function ReplyList({ id, limit, cursor }) {
+  const [replies, setReplies] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const loadReplyList = async () => {
+    setLoading(true);
+    try {
+      const replyObj = await getReplyById({ id, limit });
+      const replyList = replyObj.list;
```

Comment:
아래와 같이 작성하면 더 간결할 것 같아요! 

```suggestion
      const { list }= await getReplyById({ id, limit });
```